ninja-maven-plugin/src/main/java/ninja/build/WatchAndRestartMachine.java

```diff
@@ -219,7 +219,7 @@ public void handleNewOrModifiedFile(String newOrMod, Path path) {
         log.debug(" matched rule: type={}, pattern={}, proceed={}", match.type, match.pattern, match.proceed);
         
         if (match.proceed) {
-            log.debug(" will trigger restart", newOrMod, f);
```

Comment:
`newOrMod` and `f` are already logged in line 215 (`log.debug("{} file detected: {}", newOrMod, f);`).